.github/workflows/build_linux_elf.yml

```diff
@@ -13,7 +13,11 @@ jobs:
     - uses: actions/setup-python@v2
     - run: sudo apt-get install libsdl2-2.0-0
     - run: python -m pip install pyinstaller
+    - env:
+        URL: https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-18.04
+      run: python -m pip install --find-links ${URL} wxPython
     - run: python setup.py clean bundle
+    - run: chmod +x /home/runner/work/PythonTurtle/PythonTurtle/dist/PythonTurtle
```

Comment:
I took a look at the documentation. The [permission will be lost](https://github.com/marketplace/actions/upload-a-build-artifact#permission-loss)  in any case. The only solution is to [tar up the file](https://github.com/marketplace/actions/upload-a-build-artifact#maintaining-file-permissions-and-case-sensitive-files) instead.

All in all, I'm not happy with the zipp'ed binary. It should be possible to download the binary directly. Also, the [Upload-Artifact](https://github.com/marketplace/actions/upload-a-build-artifact) is probably the wrong Action. There should be a [Create-Release](https://github.com/actions/create-release) action that does what we need, instead.